### ISSUE TITLE
Add additional notice to /councils contact page

### ISSUE THREAD

**Issue by BenJam** [mysociety/fixmystreet#1553]
We’ve had a few of these requests in the past few days. I think we need to add something to say ‘if you wish to report a problem please click here’ to take them to the main screen or possibly to ask for their location and jump into the process from there.

**Comment by abimysoc** [comment]
@davea came up with a theory about this.

- Google e.g. "FixMyStreet Southampton"
- end up here https://www.fixmystreet.com/reports/Southampton
- no "click map to report a problem" instruction
- OK, let's try the most obvious link, which is "FixMyStreet is now available for local council websites. Find out more."
- end up on councils page
- hmm, how to report a problem here?
- there's a form!

So this ticket is a sticking plaster over the root cause, which is #1519 again on a different kind of page.

**Comment by dracos** [comment]
The old councils page did have such a notice see e.g. https://github.com/mysociety/fixmystreet/commit/3d317877844973587766b6788a462bac16543f40#diff-8395aa62704ed569217267e04ef6add5L9

**Comment by zarino** [comment]
Analytics between 12 Sept 2016 – 12 Oct 2016:

0.87% of visitors to FMS.com over that period came saw the `/about/council` page as part of their visit. Looking at these users specifically, these are the pages that their visits began on:

![screen shot 2016-10-13 at 15 11 18](https://cloud.githubusercontent.com/assets/739624/19352521/f27e3a06-9157-11e6-8ab0-8ed170ff4284.png)

Roughly a 30% come direct to `/about/council`, and 20% from the homepage (the “Are you from a council?” thing near the footer?).

The remaining 50% is a long tail of visits beginning on either council area pages (so potentially via the “now available for local council websites” banner @abimysoc and @davea noticed above) or individual report pages (likely via the “Reported […] using FixMyStreet for Councils” appended to the date/time on cobrand-generated reports).

(I've tried reducing down to sessions that included both a visit to `/about/council` and to `/contact/submit`—the page you see after you submit the contact form—but it’s only 10 visitors, 7 of which came via the homepage or `/about/council` directly, so there’s just not enough sessions remaining to draw any conclusions.)

I think there are two things at play:

* As @abimysoc and @davea noticed, there is no prominent "report a problem in this area" CTA on the `/reports/<council>` page (most likely, the big yellow banner is distracting people from the "Report a problem" button in the header).

   ![screen shot 2016-10-13 at 15 18 35](https://cloud.githubusercontent.com/assets/739624/19352606/514e2d84-9158-11e6-9306-16c00b6681c5.png)

* There is no prominent "report a problem near here" CTA on an individual report page – again, aside from the yellow "Report a problem" button.

   ![screen shot 2016-10-13 at 15 19 13](https://cloud.githubusercontent.com/assets/739624/19352633/67fe6134-9158-11e6-9ccc-6e9a52d43aab.png)

**Comment by wrightmartin** [comment]
I don't adding a notice to the /councils form will change this - they're not reading the contents of any other messages, which is why they end up there.

We (Martino) think we have a solution...

**Comment by zarino** [comment]
Fun fact! The yellow "FixMyStreet is now available for local councils" banner was removed in 881fc519900cfc2a5675925c8ac7e5080435db42.

**Comment by zarino** [comment]
What we want to avoid is something clumsy like this – It’ll inconvenience business users, and most likely just be ignored by local residents.

![screen shot 2017-02-16 at 13 13 27](https://cloud.githubusercontent.com/assets/739624/23022615/bcd77750-f449-11e6-8c88-9928a7d06bfc.png)

Looking at the problem report emails that have been erroneously submitted via the `/councils` contact form since October 2016, a few things stand out:

* The user pretty much always picks “Simple fault reporting” as the “Product”.
* They tend to type something like “Householder” or “Local resident” as their “Organisation” – or they’ll sometimes put the report category in there, like “Paving slabs”.

![screen shot 2017-02-16 at 13 04 45](https://cloud.githubusercontent.com/assets/739624/23022411/899f8da6-f448-11e6-9dda-9dcb7e5a71eb.png)

I would wager that the “Simple fault reporting” label is reinforcing their mistaken impression that this is a form for reporting problems. ie: **“Look! It says I can use this to report simple faults! Great!”** – Can we rename this product across the whole page, from “Simple fault reporting” to “Basic FixMyStreet integration” or something? Something that very clearly *isn’t* about reporting problems?

Can we add the prices to the “Product” radio buttons? Again, to reinforce the idea that these things *aren’t meant for you*. And maybe reword the “Product” form label to something more descriptive…?

![screen shot 2017-02-16 at 13 01 44](https://cloud.githubusercontent.com/assets/739624/23022333/263a6a42-f448-11e6-9ec4-a905f68acd14.png)

There’s a chance, even with the changes to these radio buttons, people will still mistakenly start filling their contact details into the form, and by the time they’ve got to the radio buttons, even though something doesn’t feel right, they’ll carry on regardless, because they’ve already invested time into filling the form out. Can we move the radio buttons to the top of the form, to make this less likely?

I also wonder whether there’s something clever we can do with the “Organisation” input – like, display a **“Oops, are you trying to report a problem in your area?”** style hint, when the input contains the words “resident” or “holder”?

**Comment by abimysoc** [comment]
I like "We are interested in ordering..." + prices. If you want to report a pothole and see that, you will definitely realise this form is not for you. I'm cool with moving the radio buttons to the top of the form too - it flows quite nicely from the section above, where the different products and prices have just been outlined.

**Comment by MyfanwyNixon** [comment]
Maybe an additional bit of labeling in the pink box - after all, you don't want things to swing the other way and have the *councils* confused that they're on the wrong page. 

Could be as simple as 'oops' before "are you trying to report a problem in your area?"

**Comment by zarino** [comment]
@MyfanwyNixon: Maybe I should have made myself clearer; when I said “What we want to avoid is something clumsy like this”, I really meant, “Under no circumstances do we want to add a pink box like this to the top of the form” ;-)

**Comment by MyfanwyNixon** [comment]
I am deleting my previous comment because I didn't read the example properly!

**Comment by zarino** [comment]
Pull request here: https://github.com/mysociety/fixmystreet/pull/1642
